src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -773,10 +773,17 @@ private bool CheckEventValueKind(BoundEventAccess boundEvent, BindValueKind valu
 
                 if (receiver?.Kind == BoundKind.BaseReference && eventSymbol.IsAbstract)
                 {
-                    Error(diagnostics, ErrorCode.ERR_AbstractBaseCall, boundEvent.Syntax, eventSymbol);
-                    return false;
+                    var baseReference = (BoundBaseReference)receiver;
+
+                    if (!eventSymbol.IsImplementableInterfaceMember() ||
+                        baseReference.ExplicitBaseReferenceOpt?.Type.IsInterfaceType() != true)
+                    {
+                        Error(diagnostics, ErrorCode.ERR_AbstractBaseCall, boundEvent.Syntax, eventSymbol);
+                        return false;
```

Comment:
I don't think this error is triggered (though it should be) for an abstract event in an interface that is `base(I)` invoked from a derived interface.

Reply:
> I don't think this error is triggered (though it should be) for an abstract event in an interface that is base(I) invoked from a derived interface.

This code path is for legacy scenarios, not when an abstract  event is in an interface. Interface case is handled in ```BindEventAssignment```, can be observed in unit-tests ExplicitBase_122 and ExplicitBase_145, for example. 



---
In reply to: [255806868](https://github.com/dotnet/roslyn/pull/33281#discussion_r255806868) [](ancestors = 255806868)